### DISCUSSION TITLE
Official plugin for Svelte and SveltKit

### DISCUSSION THREAD

**Discussion by algoflows** [nrwl/nx#22637] [Feature Requests]
# Official support for Svelte and Sveltekit

### Description
Would love Nrwl to support and deliver an officially supported Svelte plugin.

### Motivation
Svelte is the next best thing after React & Solid.

This would complete the inventory of prominent offically supported frameworks by NX at least the main frameworks out in the wild today.

### Suggested Implementation
- Offical support
- Vite (Svelte and Sveltkit already use vite)
- Vitest or Jest option
- Storybook support
- Tailwind setup automation option
- Cypress or playwrite scaffold option

### Alternate Implementations
- Offical Solildjs support 😆

**Comment by paviln** [comment]
+1 svelte is awesome

**Comment by nonameolsson** [comment]
Looking forward to this. It is a blocker for us right now to move some projects into NX.

**Comment by simpros** [comment]
I would love to see that!

**Comment by jaydeep987** [comment]
after official support for vite, was expecting official support for svelte, but sadly it didn't arrive! (but vue arrived 🤷 )

**Comment by algoflows** [comment]
@brian-nrwl what you saying, any movement or thoughts on this? SvelteKit is established now touted as the prefered way to build svelte apps, its well integrated with Vercel and is being used commercially by many many companies. Would expect sveltkit to be added to the officially supported apps....

**Comment by algoflows** [comment]
@AgentEnder hitting up the busy bods at NX for any support on this... it's been a while and things have really progressed with sveltekit and industry adoption.

**Comment by algoflows** [comment]
Hopefully the NX team answer on this.... would be great to get some feedback!

**Comment by filipwiniarski** [comment]
+1 - it's time for the official SvelteKit support ❤️

**Comment by paviln** [comment]
This issue is now over a year old, but there is yet to be any response from the nrwl team. Frustrating...

- [ ] Response

**Comment by mcmxcdev** [comment]
I found https://nx.dev/showcase/example-repos/add-svelte#add-a-svelte-project, did anyone follow these steps to verify if Nx works with Svelte? Or even with SvelteKit?

We are looking into using NX with SvelteKit but looks like Turborepo would be the better pick at this point.

**Comment by Thebigbignooby** [comment]
Before finding this thread, I opened a discussion on the sveltejs repo https://github.com/sveltejs/svelte/discussions/10425
asking for this

**Comment by shaictal** [comment]
any updates on this?
Please can we get first class support for Svelte 5

**Comment by rafeyhusain** [comment]
[This already provides support for Svelte](https://github.com/nxext/nx-extensions/tree/main) beside:

[Capacitor](https://capacitorjs.com/) plugin for Nx
[Ionic & Angular](https://ionicframework.com/docs/angular/overview) plugin for Nx
[Ionic & React](https://ionicframework.com/docs/react) plugin for Nx
[Stencil](https://stenciljs.com/) plugin for Nx
[Svelte](https://svelte.dev/) plugin for Nx
[SvelteKit](https://kit.svelte.dev/) plugin for Nx
[SolidJS](https://www.solidjs.com/) plugin for Nx
[Preact](https://preactjs.com/) plugin for Nx
